manifests/params.pp

```diff
@@ -39,7 +39,7 @@
       $packagename = 'dhcp'
       $servicename = 'dhcpd'
       $root_group  = 'root'
-      if $facts['operatingsystemrelease'] =~ /^[0-6]\./ {
+      if $facts['os']['release']['full'] =~ /^[0-6]\./ {
```

Comment:
Does it make sense to use the major and compare it as a version instead of a regex?

```suggestion
      if versioncmp($facts['os']['release']['major'], '6') <= 0
```